server/sonar-web/src/main/js/apps/coding-rules/rule/custom-rule-creation-view.js

```diff
@@ -154,7 +154,7 @@ export default ModalFormView.extend({
   sendRequest (action, options) {
     this.$('.alert').addClass('hidden');
     const that = this;
-    const url = '/api/rules/' + action;
+    const url = window.baseUrl + '/api/rules/' + action;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)